vault/resource_quota_lease_count_test.go

```diff
@@ -56,6 +56,44 @@ func TestQuotaLeaseCount(t *testing.T) {
 	})
 }
 
+func TestQuotaLeaseCountWithRole(t *testing.T) {
+	name := acctest.RandomWithPrefix("lease-count")
+	backend := acctest.RandomWithPrefix("approle")
+	role := acctest.RandomWithPrefix("test-role")
+	leaseCount := "1001"
+	newLeaseCount := "2001"
+	resourceName := "vault_quota_lease_count.foobar"
+
+	resource.Test(t, resource.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testutil.TestEntPreCheck(t) },
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testQuotaLeaseCountCheckDestroy([]string{leaseCount, newLeaseCount}),
+			testAccCheckAppRoleAuthBackendRoleDestroy,
+		),
+		Steps: []resource.TestStep{
```

Comment:
Can we add an Import test step to test the import flow? See https://github.com/hashicorp/terraform-provider-vault/blob/c2648993af7cdd18da5167d1ee8c3c3934dc3ad6/vault/resource_azure_secret_backend_test.go#L146 for an example.

Reply:
Yeah, for sure! Thanks for pointing this out. I'm so new to this code, I didn't even realize this was a thing.

Reply:
Does this look right? https://github.com/hashicorp/terraform-provider-vault/pull/1994/commits/45018c9ad486319391debfa916704a4a1c73d54c